keras_cv/layers/overlapping_patching_embedding.py

```diff
@@ -0,0 +1,23 @@
+from keras_cv.backend import keras
+from keras_cv.backend import ops
+
+
+@keras.saving.register_keras_serializable(package="keras_cv")
+class OverlappingPatchingAndEmbedding(keras.layers.Layer):
+    def __init__(self, out_channels=32, patch_size=7, stride=4, **kwargs):
```

Comment:
Same here for docstring -- I think particularly the shape transformation is worth noting in the doc